src/jvm/main/org/jetbrains/kotlinx/lincheck/trace/TracePoint.kt

```diff
@@ -153,10 +153,10 @@ internal class ObstructionFreedomViolationExecutionAbortTracePoint(
 }
 
 internal class ReadTracePoint(
-    private val ownerRepresentation: String?,
+    var ownerRepresentation: String?,
```

Comment:
Let' try to keep it `private` and `val`, and apply all necessary transformations upon initialization.